source/locale/en/symbols.dic

```diff
@@ -97,7 +96,12 @@ _	line	most
 ▪	black square	some
 ◾	black square	some
 ◦	white bullet	some
-➔	right arrow	some

```

Comment:
Also, that arrow (U+2794) is not used as a bullet shape whereas the other arrows are.